docs/integrations/databases/opentelemetry/mysql-opentelemetry.md

```diff
@@ -0,0 +1,278 @@
+    -   Dashboard based on Slow Query format:
+        -   MySQL - Slow Queries
+-   MySQL General Health is based on logs from the MySQL Error log format.
+-   MySQL Replication is based on logs from the MySQL Error log format.
+-   MySQL Slow Queries is based on logs from the MySQL Slow Queries log format.
+
+The MySQL app dashboards dependent on error logs are based on the message types ERROR, NOTE, Warning, and Info. For more details on the MySQL log file format, see <http://dev.mysql.com/doc/refman/5.5/en/server-logs.html>.
+
+## Fields creation in Sumo Logic for Mysql
+
+Following are the tags which will be created as part of MySql App install if not already present. 
+
+- `db.cluster.name` - User configured. Enter a name to identify this MySql cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- `db.system` - Has fixed value of **mysql**
+- `sumo.datasource` - has fixed value of **mysql**
+- `db.node.name` - Has the value of host name of the machine which is being monitored
+
+## Prerequisites
+
+For metric collection [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/mysqlreceiver#prerequisites) are the prerequisites.
+
+For log collection 
+
+Configure MySQL to log to a local file(s). MySQL logs written to a log file can be collected via the Local File Source of a Sumo Logic Installed Collector. To configure the MySQL log file(s), locate your local my.cnf configuration file in the database directory.
+
+1.  Open `my.cnf` in a text editor.
+2.  Set the following parameters in the `[mysqld]` section:
+```sql
+[mysqld]
+    log_error = /var/log/mysql/error.log
+    slow_query_log=1
+    slow_query_log_file = /var/log/mysql/mysql-slow.log
+    long_query_time=2
+    long_query_time=2
+```
+
+[Error Logs](https://dev.mysql.com/doc/refman/5.7/en/slow-query-log.html). By default, error logs are enabled and are logged at file specified by the log_error key.
+
+[Slow Query Logs](https://dev.mysql.com/doc/refman/5.7/en/slow-query-log.html). slow_query_log=1 enables logging of slow queries to the file specified by `slow_query_log_file`. Setting `long_query_time=2` will cause queries that take more than two seconds to execute to be logged. The default value of `long_query_time` is 10 seconds.
+
+[General Query Logs](https://dev.mysql.com/doc/refman/5.7/en/query-log.html). We don't recommend enabling general_log for performance reasons. These logs are not used by the Sumo Logic MySQL App.
+
+1.  Save the `my.cnf` file.
+2.  Restart the MySQL server: `sudo mysql.server restart`
+
+## Collecting Logs, Metrics & installing App for MySQL
+
+Here are the steps for collecting logs, metric and installing the app:
+
+### Step 1 : Set up collector
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the **Use an existing Collector** option.
+:::
+
+If you want to create a new collector please select the **Add a new collector** option.
```

Comment:
```suggestion
To create a new Collector:
	
1. Select the **Add a new Collector** option.
2. Select the platform for which you want to install the Sumo OpenTelemetry Collector.
	
This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.
```